Simplenote/RecoveryUnarchiver.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import UniformTypeIdentifiers
+import CoreData
+
+public class RecoveryUnarchiver {
+    private let fileManager: FileManager
+
+    public init(fileManager: FileManager = .default) {
+        self.fileManager = fileManager
+    }
+
+    var simperium: Simperium {
+        SimplenoteAppDelegate.shared().simperium
```

Comment:
Since the AppDelegate will use this class, and it'll have access to the Simperium instance, you can get this via initializer